oam/install_scripts/mcs-loadbrm.py

```diff
@@ -1,70 +1,88 @@
-#!/usr/bin/env python3
-
-import configparser
+#!/usr/bin/env python
```

Comment:
This shebang causes issue on centos8

Reply:
I replaced it using the platform we build packages for.